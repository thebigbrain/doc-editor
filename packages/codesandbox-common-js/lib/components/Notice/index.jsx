import styled from 'styled-components';
export default styled.div `
  font-size: 0.75rem;
  color: white;
  padding: 0.125rem 0.2rem;
  background-image: linear-gradient(
    45deg,
    ${({ theme }) => theme.secondary.darken(0.2)} 0%,
    ${({ theme }) => theme.secondary.darken(0.1)} 100%
  );
  border-radius: 4px;
  float: right;
  margin-right: 2rem;
`;
