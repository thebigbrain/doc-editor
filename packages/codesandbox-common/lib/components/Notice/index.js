"use strict";

var _interopRequireDefault = require("@babel/runtime/helpers/interopRequireDefault");

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports["default"] = void 0;

var _taggedTemplateLiteral2 = _interopRequireDefault(require("@babel/runtime/helpers/taggedTemplateLiteral"));

var _styledComponents = _interopRequireDefault(require("styled-components"));

function _templateObject() {
  var data = (0, _taggedTemplateLiteral2["default"])(["\n  font-size: 0.75rem;\n  color: white;\n  padding: 0.125rem 0.2rem;\n  background-image: linear-gradient(\n    45deg,\n    ", " 0%,\n    ", " 100%\n  );\n  border-radius: 4px;\n  float: right;\n  margin-right: 2rem;\n"]);

  _templateObject = function _templateObject() {
    return data;
  };

  return data;
}

var _default = _styledComponents["default"].div(_templateObject(), function (_ref) {
  var theme = _ref.theme;
  return theme.secondary.darken(0.2);
}, function (_ref2) {
  var theme = _ref2.theme;
  return theme.secondary.darken(0.1);
});

exports["default"] = _default;