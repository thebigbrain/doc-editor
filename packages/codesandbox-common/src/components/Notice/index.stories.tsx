import React from 'react';
import { storiesOf } from '@storybook/react';
import Notice from '.';

const stories = storiesOf('components/Notice', module);

stories.add('Notice', () => <Notice>You need to Login</Notice>);
