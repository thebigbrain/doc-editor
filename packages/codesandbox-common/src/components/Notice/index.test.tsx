import 'jest-styled-components';
import React from 'react';
import mountWithTheme from '../../test/themeMount';
import Notice from '.';

describe('<Notice /> rendering', () => {
  it('basic', () => {
    const wrapper = mountWithTheme(<Notice>You need to Login</Notice>);
    expect(wrapper).toMatchSnapshot();
  });
});
