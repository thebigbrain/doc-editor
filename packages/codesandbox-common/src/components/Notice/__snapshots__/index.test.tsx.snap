// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Notice /> rendering basic 1`] = `ReactWrapper {}`;
